### DISCUSSION TITLE
Published documentation contains typo, but has been fixed in master

### DISCUSSION THREAD

**Discussion by Colin-XKL** [containrrr/watchtower#655] [Q&A] [answered]
**Describe the bug**
something wrong with the document. 
*Container selection - Watchtower*
*https://containrrr.dev/watchtower/container-selection/*
Environment variable  `WATCTOWER_LABEL_ENABLE ` is wrong. It should be "WA**TCH**" not "WA**TC**".  But when I click the edit button, I found that in the origin file at the git repository, the word is correct. So I gave up pulling a request and report it here.
**To Reproduce**
https://containrrr.dev/watchtower/container-selection/

**Screenshots**
![image.png](https://i.loli.net/2020/10/04/VEzKp3dWi1oO7NC.png)

**Comment by piksel** [comment, marked as answer]
Yeah, this is because it has been fixed, but no new release has been done since then. The documentation is supposed to always reflect the released version (which, unfortunately includes typos 😔)
